src/Integrations/Goodbits/Goodbits.php

```diff
@@ -54,7 +54,7 @@ class Goodbits extends AbstractFormBuilder implements MapperInterface, ServiceIn
 	 *
 	 * @var ValidatorInterface
 	 */
-	public $validator;
+	protected $validator;
```

Comment:
This doesn't seem to be coming from the `AbstractFormBuilder` but is a validator injection, which means its visibility can be `private`.